[EXC-3637] Adding version agnostic documentation (#55)
mulesoft/docs-exchange: modules/ROOT/pages/to-publish-assets-maven.adoc

+
*Notes:*
+
** *Important:* The `<classifier>` element must be set to make the asset visible in Exchange.

> ** The `name` property represents the visible name of the asset. It only be used when the asset has the first version of a combination of a GroupId and an artifactId. If this property must be changed, it should be done from the asset portal: see xref:to-describe-an-asset.adoc[To Describe an Asset] for details.

Reviewer: ```suggestion
** The `name` property represents the visible name of the asset. It only be used when the asset has the first version of a combination of a GroupId and an artifactId. To change it, go to the asset portal: see xref:to-describe-an-asset.adoc[To Describe an Asset] for details.
```

> ** The `description` property represents the description of the asset. It only be used when the asset has the first version of a combination of a GroupId and an artifactId. If this property must be changed, it should be done from the asset portal: see xref:to-describe-an-asset.adoc[To Describe an Asset] for details.

Reviewer: ```suggestion
** The `description` property represents the description of the asset. It only be used when the asset has the first version of a combination of a GroupId and an artifactId. To change it, go to the asset portal: see xref:to-describe-an-asset.adoc[To Describe an Asset] for details.
```
